### PR TITLE
Added PAI calculation from Massman and namelist option.

### PR DESCRIPTION
**Pull request by drnimbusrain**

### CODE REVIEW

**Comment by zmoon** [comment]
`fixpai` descriptions still say default false in some places

**Review by zmoon — COMMENTED** [review]
Is there a reason to *not* use `fixpai`? If not, could just remove the other option. Is there a possibility of adding an additional option for this PAI calculation? In that case, could represent with integer instead.

File: canopy_parm_mod.F90 [hunk 1]
```diff
@@ -56,15 +58,24 @@ SUBROUTINE CANOPY_PARM( VTYPE, FCH, FFRAC, LAI, &
...
+        !approx/average vegtype mapping to Massman et al. forest types 
         if (VTYPE .ge. 1 .and. VTYPE .le. 2) then !VIIRS Cat 1-2/Evergreen Needleleaf & Broadleaf 
                                                   !--> Use average Massman Aspen+Spruce+Pine Forest
          FIRETYPE=0
          CDRAG=(0.20_rk + 0.25_rk + 0.20_rk + 0.20_rk + 0.20_rk)/5.0_rk
-         PAI=(5.73_rk + 3.28_rk + 2.41_rk + 2.14_rk + 3.78_rk)/5.0_rk  !will use Massman calculation (Eq. 19) of PAI later...
+         if (FIXPAI) then
+            PAI=(5.73_rk + 3.28_rk + 2.41_rk + 2.14_rk + 3.78_rk)/5.0_rk
+         else
+            PAI=( (FCH*(FFRAC/3.0_rk)*10.6955_rk) / (2.0_rk * pi) ) * FFRAC !Massman PAI calculation (Eq. 19)
```
[the remark this answers is not in the document]
Comment (drnimbusrain) [reply canopy_parm_mod.F90:73]: Yes, I agree to make a function.

**Review by drnimbusrain — COMMENTED** [review]
@zmoon Thank you for the review.  Thanks for catching the fixpai=true issue, and for recommending using a function.  To answer your question, yes, the Plant Area Index (PAI) may not be fixed on vegtype and can be calculated as a function of grid cell canopy height and forest/canopy fraction (eq. 19 in Massman et al).  PAI (similar but a fundamental different parameter than LAI) is a significant parameter in Massman and in paramterizing the shape of the canopy distribution.  There is also much uncertainty in the calculation, and maybe some uncertainity in my interpretation.  Also, there may be other ways to calculate it as a function of LAI, which would be optimal as that is the fundamental quantity we can take from the gridded met/land model in UFS.  I have not found this relationship yet, but would really appreciate the help here.

**Comment by zmoon** [comment]
Thanks for the explanation. I was confused in my question, I suppose I meant to ask why use the fixed one. But I think it makes sense to keep both. If there might be more methods added, could switch now to integer to represent PAI method choice.

> there may be other ways to calculate it as a function of LAI, which would be optimal as that is the fundamental quantity we can take from the gridded met/land model in UFS

Yeah, I imagine there should be some scaling factors based on plant type. I'll see if I find something. Also the land model should be using PAI internally at least for something, maybe we could get it.

**Comment by zmoon** [comment]
Another thought is that some of the argument lists are getting pretty long. We could introduce some types soon. Maybe first one to hold the config settings.

**Review by zmoon — APPROVED** [review]

**Pull request merged**